rest/client.py

```diff
@@ -44,8 +56,21 @@ def order_book(self, currency_pair: str, group: int = 1):
         if group not in [0, 1, 2]:
             raise ValueError('Group parameter should be 0, 1 or 2.')
 
-        rsp = requests.get(url=self.base_endpoint + '/order_book/' + currency_pair, params={'group': group})
-        if rsp.status_code == 404:
-            raise ValueError('Order book for requested currency pair not found.')
+        data = {'group': group}
```

Comment:
Again, this isn't data, its params. data should be body of the request that we're going to use in more complex requests.